internal/hydraclient/hydraclient.go

```diff
@@ -0,0 +1,51 @@
+package hydraclient
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gofrs/uuid"
+
+	"github.com/ory/herodot"
+	hydraclientgo "github.com/ory/hydra-client-go"
+	"github.com/ory/kratos/session"
+)
+
+func GetHydraAdminAPIClient(hydra_admin_url string) hydraclientgo.AdminApi {
+	configuration := hydraclientgo.NewConfiguration()
+	configuration.Servers = hydraclientgo.ServerConfigurations{{
+		URL: hydra_admin_url,
+	}}
+	return hydraclientgo.NewAPIClient(configuration).AdminApi
+}
+
+func GetHydraLoginRequest(hydra_admin_url string, hlc uuid.UUID) (*hydraclientgo.LoginRequest, error) {
```

Comment:
Go code does not use underscores :) Please change all variable names that have underscores!